src/lib/components/ProviderLogin/provider-login.container.js

```diff
@@ -97,7 +102,7 @@ export default class LoginComponent extends Component<Props> {
         onError(error);
       }
       // Show form error messsage when idp is null
-      if (error.name !== 'unknown') {
+      if (error.name === 'unknown') {
```

Comment:
It seems that this might only show an error if the name is 'unknown'? If we want to show all errors maybe we can remove the conditional.